### PR TITLE
Updates for the new Symfony CS

### PR DESCRIPTION
**Pull request by GrahamCampbell**
This PR is a follow-up to an email conversation RE changes to the Symfony preset (following changes to the Symfony style guide itself).

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
I think these resent test failures are likely related to changes in the php-parser package. I've not looked in detail, but I suspect the issue started around PHP-Parser 4.6 or 4.7.

**Comment by bobthecow** [comment]
The current failures are symfony/symfony#37674, a regression in VarDumper v5.1.3. It should be fixed now, but apparently not released?

File: .styleci.yml [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
+  - native_constant_invocation_symfony
+  - native_function_invocation_symfony
+  - no_superfluous_phpdoc_tags_symfony
+  - pow_to_exponentiation
   - pre_increment
   - unalign_double_arrow
   - unalign_equals
   - yoda_style
-  - property_separation
-  - const_separation
```
Comment (bobthecow) [inline .styleci.yml:22]: are there any more overrides we should remove with this new batch of preset changes?

Comment (GrahamCampbell) [reply .styleci.yml:22]: No. If there were, StyleCI would fail loudly that a fixer that was not enabled by the preset cannot be disabled. :)

Comment (bobthecow) [reply .styleci.yml:22]: oh, no, i meant whether we should remove any overrides to get closer to the base preset, not to keep styleci from whining 😉

Comment (GrahamCampbell) [reply .styleci.yml:22]: Oh, I guess pre_increment could go. I don't think there'd be too many changes from that. The others will cause quite a lot of changes, however. There was also the discussion we had about the slashes on internal constants and functions. You could un-disable the two _symfony fixers listed here, or keep them disabled and enable the full versions.

Comment (bobthecow) [reply .styleci.yml:22]: let's leave `pre_increment`. that, along with `yoda_style`, is one of those things where symphony's coding standard makes you think about things backwards in the name of (questionable) protection from yourself.

looking at overlap between laravel, symfony, and styleci's "recommended", it seems like i'm pretty clearly in the minority for `concat_without_spaces`, `method_argument_space` and `unalign_equals`. we can remove those overrides 😢

Comment (bobthecow) [reply .styleci.yml:22]: it might be worth switching the big ones (spaces and alignment changes) in their own diff to keep things cleaner.

**Pull request merged**